### PR TITLE
Adds latest version of Veterans Crisis Line buttons.

### PR DESCRIPTION
**Pull request by webinista**

### CODE REVIEW

File: lib/va_common/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module VaCommon
-  VERSION = "0.4.0"
+  VERSION = "0.5.0"
```
Comment (ayaleloehr) [inline lib/va_common/version.rb]: If you didn't actually do a deploy to rubygems.org yet, I believe you can just leave this as 0.4.0 still until the next deploy?

File: app/assets/stylesheets/components/_va-crisis-line.scss [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+}
+
+//===== VA CRISIS LINE BUTTONS 
+.va-crisis-line--touch {
+  border-radius: .2em;
+  margin: 2.5em 0;
+}
+
+.va-crisis-line--notouch {
+  background: $color-gold url(/assets/images/arrow-up.svg) no-repeat 97% center;
```
Comment (ayaleloehr) [inline app/assets/stylesheets/components/_va-crisis-line.scss]: I don't believe this file exists in this repo?

File: app/assets/stylesheets/components/_va-crisis-line.scss [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  background-size: .8em auto;
+  border-radius: .2em .2em 0 0;
+  font-size: 1.7rem;
+  margin: 0;
+  padding: 1.25rem;
+  text-align: left;
+  width: 100%;
+  
+  .va-crisis-panel--open & {
+    background-image: url(/assets/images/arrow-down.svg);
```
Comment (ayaleloehr) [inline app/assets/stylesheets/components/_va-crisis-line.scss]: Same as above comment

**Comment by ayaleloehr** [comment]
3 comments but LGTM other than that.  Feel free to merge once those 3 are addressed (or assign back to me if you prefer).

File: app/assets/stylesheets/components/_va-crisis-line.scss (hunk 2 above)
Comment (ayaleloehr) [inline app/assets/stylesheets/components/_va-crisis-line.scss]: @webinista This path (and the one below it) doesn't match the directory where the file is now. Will it automagically find it or is this a typo?

**Comment by webinista** [comment]
Merging this PR so that I can test it with a project.

**Pull request merged**